app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -26,37 +27,51 @@ public Item(int id, boolean deleted, String type, String by, long time, String t
         this.dead = dead;
     }
 
-    /** The item's unique ID */
+    /**
+     * The item's unique ID
```

Comment:
Hemm...this is interesting. I didn't realize the method's name is quite self-explanatory. When I made the accessor / mutator comments, I thought about generating JavaDoc. Then again, I don't think it's necessary for now.

OK.